codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecoderTest.java

```diff
@@ -159,6 +184,70 @@ public void testResponseDecompression() {
         assertFalse(channel.finish()); // assert that no messages are left in channel
     }
 
+    @Test
+    public void testResponseBrotliDecompression() {
+        HttpResponseDecoder decoder = new HttpResponseDecoder();
+        HttpContentDecoder decompressor = new HttpContentDecompressor();
+        HttpObjectAggregator aggregator = new HttpObjectAggregator(Integer.MAX_VALUE);
+        EmbeddedChannel channel = new EmbeddedChannel(decoder, decompressor, aggregator);
+
+        String headers = "HTTP/1.1 200 OK\r\n" +
+          "Content-Length: " + SAMPLE_BZ_BYTES.length + "\r\n" +
+          "Content-Encoding: br\r\n" +
+          "\r\n";
+        ByteBuf buf = Unpooled.wrappedBuffer(headers.getBytes(CharsetUtil.US_ASCII), SAMPLE_BZ_BYTES);
+        assertTrue(channel.writeInbound(buf));
+
+        Object o = channel.readInbound();
+        assertThat(o, is(instanceOf(FullHttpResponse.class)));
+        FullHttpResponse resp = (FullHttpResponse) o;
+        assertEquals(SAMPLE_STRING.length(), resp.headers().getInt(HttpHeaderNames.CONTENT_LENGTH).intValue());
+        assertEquals(SAMPLE_STRING, resp.content().toString(CharsetUtil.UTF_8));
+        resp.release();
+
+        assertHasInboundMessages(channel, false);
+        assertHasOutboundMessages(channel, false);
+        assertFalse(channel.finish()); // assert that no messages are left in channel
+    }
+
+    @Test
+    public void testResponseChunksBrotliDecompression() {
```

Comment:
Thanks for adding tests for HTTP. Can you please also add a `BrotliDecoderTest` as a subclass of `AbstractDecoderTest`? It will run the common tests we have for other decoders and you can add more brotli-specific tests there if necessary.

Reply:
done